packages/db/.env.example

```diff
@@ -1,2 +1,2 @@
 
-DATABASE_URL="postgresql://postgres:mysecretpassword@localhost:5432/postgres"
+DATABASE_URL="postgresql://neondb_owner:kUVepI07lrXL@ep-sparkling-resonance-a59z7i74-pooler.us-east-2.aws.neon.tech/neondb?sslmode=require"
```

Comment:
hey, I think you exposed your db url

Reply:
yaa!! i know , i already deleted that after work , Thanks for mentioning 👍